### ISSUE TITLE
Mod Conflict causing button overlaps (1.19.2)

### ISSUE THREAD

**Issue by ScriptKittyJane** [Lizard-Of-Oz/Inventorio#247]
I'm playing on 1.19.2 and having some conflict issues with other UI mods like [Cosmetic Armor Reworked](https://www.curseforge.com/minecraft/mc-mods/cosmetic-armor-reworked), Buttons keep overlapping each other like in the images below. If possible, a config option to change the position of the toggle button would be appreciated.

![2023-07-17_15 49 51](https://github.com/Lizard-Of-Oz/Inventorio/assets/94088381/a42ee810-13b6-4671-aace-0d0ef7c5bfcc)
![2023-07-17_15 49 43](https://github.com/Lizard-Of-Oz/Inventorio/assets/94088381/f7144f99-4ea6-4e78-9b24-0d7e5769fe88)

**Comment by Jpssz** [comment]
Yeah! i agree with you! i cant use the creative inventory features because the button is under the tab switch!
![Captura de tela 2023-08-09 194943](https://github.com/Lizard-Of-Oz/Inventorio/assets/141881338/9d1f53f2-a86a-4311-8e99-43af412e07d8)
